### DISCUSSION TITLE
Cypress forced me to update to 10.4.0

### DISCUSSION THREAD

**Discussion by piotrpog** [cypress-io/cypress#23276] [General]
Today when i tried to run cypress, it tried to force me to update to latest version:

```
> npx cypress open
Need to install the following packages:
  cypress@10.4.0
Ok to proceed? (y)
```
Is there any way to still use old version? I did not updated node, npm or `package.json` in any way which could cause this forced update.

**Comment by stavares843** [comment]
hello! 👋 

if you revert the changes you can still use the previous version

**Reply by piotrpog** [reply]
> hello! 👋
> 
> if you revert the changes you can still use the previous version

`package.json` was not changed in any way, so there is nothing to revert. When i use earlier version it just displays that i need to install 10.4.0

**Comment by emilyrohrbough** [comment]
@piotrpog this is a side-effect of npx. Please read: https://docs.npmjs.com/cli/v7/commands/npx

try `npx cypress@<insert_version> open`
